### PR TITLE
Disable num_workers>0 at OTE CI test

### PR DESCRIPTION
**Pull request by sungmanc**
Reflect latest commits

### CODE REVIEW

**Comment by sungmanc** [comment]
ote run-test

File: external/model-preparation-algorithm/configs/classification/configuration.yaml [hunk 1]
```diff
@@ -82,7 +82,7 @@ learning_parameters:
     visible_in_ui: true
     warning: null
   num_workers:
-    affects_outcome_of: NONE
+    affects_outcome_of: TRAINING
```
Comment (goodsong81) [inline external/model-preparation-algorithm/configs/classification/configuration.yaml:85]: Speed changes but the output does not.
@harimkang What do you think?

Comment (harimkang) [reply external/model-preparation-algorithm/configs/classification/configuration.yaml:85]: What does this variable do? Does that mean it's only used for training?

Comment (sungmanc) [reply external/model-preparation-algorithm/configs/classification/configuration.yaml:85]: > Speed changes but the output does not. @harimkang What do you think?

Hmm, I can't understand this sentence. what's the meaning of the output?

Comment (sungmanc) [reply external/model-preparation-algorithm/configs/classification/configuration.yaml:85]: > What does this variable do? Does that mean it's only used for training?

That field is required to `hyperparmaeter_override`. If `affects_outcome_of` is set to `NONE`, that variable is not overridden.

image

Comment (goodsong81) [reply external/model-preparation-algorithm/configs/classification/configuration.yaml:85]: Output of training = model (accuracy)

Comment (goodsong81) [reply external/model-preparation-algorithm/configs/classification/configuration.yaml:85]: It seems that 'learning rate' like h-params are marke as 'affects_outcome_of: TRAINING'.

Comment (sungmanc) [reply external/model-preparation-algorithm/configs/classification/configuration.yaml:85]: Yeah, if we want to override a specific hyper-parameter defined on `configuration` (i.e. `learning rate`), then that variable should be set as `affects_outcome_of = TRAINING or INFERENCE`. 

So, in this case, because I want to override `num_workers` variable, I changed the `affects_outcome_of = TRAINING`. And add additional argument `learning_parameters.num_workers=0` for passing the OTE CI test.

However, as @harimkang said in PR #1367, if there is no problem on `num_workers`, then let's drop this PR

**Comment by sungmanc** [comment]
@goodsong81 @harimkang , I have tried 3 trials CI tests, and there is no `num_workers` related FAILED cases.
So, I closed this PR. 

http://validationreports.sclab.intel.com:8006/reports/build_number_report?test_session_build_number=pr-ote-test-740&environment=iotg-dev-workstation-09
http://validationreports.sclab.intel.com:8006/reports/build_number_report?test_session_build_number=pr-ote-test-738&environment=iotg-dev-workstation-12
The result of one CI test case was weird because the number of tests was quite small. However, even in this case, there is no `num_workers` related FAIL case
http://validationreports.sclab.intel.com:8006/reports/build_number_report?test_session_build_number=pr-ote-test-739&environment=iotg-dev-workstation-10

**Comment by sungmanc** [comment]
run ote-test

**Comment by sungmanc** [comment]
run ote-test

**Comment by sungmanc** [comment]
run ote-test

**Comment by sungmanc** [comment]
run ote-test

**Comment by sungmanc** [comment]
run ote-test

**Comment by sungmanc** [comment]
run ote-test

**Comment by sungmanc** [comment]
run ote-test

**Comment by sungmanc** [comment]
We'll use num_workers > 0 for > OTEv0.4.0 and also will handle the stability issue by using a kind of auto-config.
So, I'm going to close this PR
